modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_set.scss

```diff
@@ -50,78 +50,145 @@ body {
 
 // Buttons
 
+.btn-danger {
```

Comment:
I prefer to group `btn-danger`, `btn-info`, `btn-dark`, etc together and `btn-outline-*` selectors together. They're separate modifiers and it's easier to find imo.